src/core/integrators/velocity_verlet_npt.cpp

```diff
@@ -0,0 +1,193 @@
+
+#include "config.hpp"
+
+#ifdef NPT
+#include "ParticleRange.hpp"
+#include "cells.hpp"
+#include "communication.hpp"
+#include "grid.hpp"
+#include "integrate.hpp"
+#include "nonbonded_interactions/nonbonded_interaction_data.hpp"
+#include "npt.hpp"
+#include "particle_data.hpp"
+#include "thermostat.hpp"
+#include "utils/math/sqr.hpp"
+
+void velocity_verlet_npt_propagate_vel_final(const ParticleRange &particles) {
```

Comment:
it would also be nice to have a docstring mentioning which part of VV is performed here (maybe a reference to literature)